packages/frontend/src/hooks/useQueryResults.tsx

```diff
@@ -67,19 +33,32 @@ export const useQueryResults = ({
             query: MetricQuery;
         }
     >(getQueryResults, {
-        mutationKey: ['queryResults', tableId, metricQuery, projectUuid],
+        mutationKey: ['queryResults', projectUuid],
```

Comment:
Is there any benefit to keeping `projectUuid` ?

Reply:
Not really, I can remove it as well.